scripts/example-mc.jl

```diff
@@ -0,0 +1,77 @@
+# Define parameters for MCMC simulation
+L = square_supercell_dimensions[1]
+M = square_supercell_dimensions[2]
+Ns = [4]
+adsorption_energy = -0.04
+nn_energy = -0.01
+nnn_energy = -0.0025
+temperatures = 200:-1:1
+num_equilibration_steps = 10_000
+num_sampling_steps = 10_000
+random_seed = 1234
+k_B = 8.617_333_262e-5
```

Comment:
@yangmr04: Can we hard-code this (NIST's) value for k_B (in eV/K) into FreeBird.jl?

Reply:
Yup, can be done by declaring a constant with `const k_B` in FreeBird.